websites/P/Puray Animes/presence.ts

```diff
@@ -0,0 +1,61 @@
+const presence = new Presence({
+		clientId: "972246349917610054"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const defaultData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname } = window.location,
+		data: PresenceData = defaultData;
+	if (document.querySelector("input[id^=headlessui]"))
+		data.details = "🔍・Pesquisando Animes";
+	else if (pathname.startsWith("/home")) {
+		data.details = "Início:";
+		data.state = "Visualizando animes.";
+		data.buttons = [{ label: "💻・Puray Animes", url: location.href }];
+	} else if (pathname.startsWith("/profile/")) {
+		const username = document.querySelector("h3").childNodes[0]?.textContent;
+		data.details = "Visualizando Perfil:";
+		data.state = `${username} - ${
+			document.querySelector("button[class*=blue] span")?.textContent
+		}`;
+		data.startTimestamp = browsingTimestamp;
+		data.buttons = [{ label: `🐒・Perfil de ${username}`, url: location.href }];
```

Comment:
Buttons should not repeat the data shown above
```suggestion
		data.buttons = [{ label: `Ver perfil`, url: location.href }];
```